modules/ROOT/pages/basketball-robot-guide/mechanics/milling.adoc

```diff
@@ -0,0 +1,430 @@
+
+Activating FEED HOLD does not stop the spindle.
+If FEED HOLD is activated when the tool is inside the material,
+the tool can keep spinning and melt itself stuck to the material, which can damage the spindle.
+====
+
+IMPORTANT:
+====
+btn:[Spindle] button must only be pressed when pausing the cycle.
+
+Do not press the spindle button at any other time.
+====
+
+==== 7 - Tool Length Sensor (TLS)
+
+The btn:[TLS] button is used together with the physical Tool Length Sensor to set the height of your tool.
+
+
+==== 8 - CYCLE START
+
+Clicking btn:[CYCLE START] will run the loaded in G-code.
+
+
+==== 9 - LOAD FILE
+
+btn:[LOAD FILE] lets you load in your G-code file.
+
+
+==== 10 - Command Line
+
+This field lets you manually enter machine code.
+
+
+==== 11 - CYCLE STOP
+
+Clicking btn:[CYCLE STOP] will stop the current cycle.
+When the loaded in G-code finishes, the machine will automatically stop.
+
+This button should only be used if something unexpected happens during the cycle.
+
+
+==== 12 - Reset
+
+The btn:[RESET] button is used to set and release the machine from the reset state.
+The machine cannot be moved when the machien is in reset state and no motion can be executed.
+
+btn:[RESET] blinks when the machine is in reset and goes out of reset when clicked.
+
+
+=== Jogging
+
+
+==== Jog panel
+
+image::mill_uccnc_jog_panel.png[]
+
+Moving the mouse pointer to the left side of UCCNC will open the jog panel.
+
+Jogging is moving the machine manually outside of work processes.
+
+The axis have *+* and *–* jog buttons on the jog panel, pressing these buttons, all axis can be jogged to their
+negative and positive directions.
+
+With btn:[Continuous] active, the machine axis jogs while the jog button of the axis is being pressed and
+the jog finishes when the button gets released.
+
+With btn:[Step] active, the axis moves the selected distance for every jog button presses.
+Currently there are 0.001, 0.01, 0.1 and 1 unit lengths are selectable. +
+Unit lengths moved in Step mode can be selected with the btn:[0.001], btn:[0.01], btn:[0.1] and btn:[1] buttons.
+
+The *Jog feed* sets the feedrate of the jog movements.
+
+
+==== Jogging with keyboard
+
+The machine's axis can also be jogged through the keyboard.
+
+kbd:[↑] and kbd:[↓] jog the Y-axis.
+
+kbd:[←] and kbd:[→] jog the X-axis.
+
+kbd:[PgUp] and kbd:[PgDn] jog the Z-axis.
+
+
+=== Machine XYZ-coordinates
+
+image::mill_stepcraft_xyz.png[]
+
+The machine's XYZ-axis directions are shown in the image.
+The origin point of the machine is also near here.
+When homing, the machine moves the spindle to the end of the Y-axis.
+This is done so there is better access to the table of the machine.
+
+
+
+== Before milling
+
+=== 1. Inspect the machine
+
+Check if any of the rail covers are open, if there are any material pieces on the table,
+if everything looks the way it should.
+
+
+=== 2. Clean the table to remove debris

```

Comment:
I would add the requirement to check the spindle for debris. Not the first time someone cooks the bearings.